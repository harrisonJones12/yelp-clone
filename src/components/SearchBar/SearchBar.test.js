import React from 'react';
import SearchBar from './SearchBar';
import renderer from 'react-test-renderer';

it('it renders correctly', () => {
    const tree = renderer
    .create(<SearchBar></SearchBar>)
    .toJSON();
    expect(tree).toMatchSnapShot();
});