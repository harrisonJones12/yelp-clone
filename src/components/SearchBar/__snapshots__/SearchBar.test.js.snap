// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly 1`] = `
<div
  className="SearchBar"
>
  <div
    className="SearchBar-sort-options"
  >
    <ul>
      <li
        className="active"
        onClick={[Function]}
      >
        Best Match
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Highest Rated
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Most Reviewed
      </li>
    </ul>
  </div>
  <div
    className="SearchBar-fields"
  >
    <input
      onChange={[Function]}
      placeholder="Search Businesses"
    />
    <input
      onChange={[Function]}
      placeholder="Where?"
    />
  </div>
  <div
    className="SearchBar-submit"
  >
    <a
      onClick={[Function]}
    >
      Let's Go
    </a>
  </div>
</div>
`;
